lib/make-disk-image.nix

```diff
@@ -15,10 +18,14 @@ let
   };
   cleanedConfig = diskoLib.testLib.prepareDiskoConfig nixosConfig.config diskoLib.testLib.devices;
   systemToInstall = nixosConfig.extendModules {
-    modules = [{
-      disko.devices = lib.mkForce cleanedConfig.disko.devices;
-      boot.loader.grub.devices = lib.mkForce cleanedConfig.boot.loader.grub.devices;
-    }];
+    modules = [
+      extraConfig
+      {
+        disko.testMode = true;
```

Comment:
We don't need testMode if we can override the password file in luks.

Reply:
password cannot be overwritten, so testMode is the easiest way to fix that